api/src/main/java/org/openmrs/module/attachments/obs/ComplexViewHelper.java

```diff
@@ -0,0 +1,15 @@
+package org.openmrs.module.attachments.obs;
+
+/**
+ * The contents of this file are subject to the OpenMRS Public License Version 1.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy of the
+ * License at http://license.openmrs.org Software distributed under the License is distributed on an
+ * "AS IS" basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+ * specific language governing rights and limitations under the License. Copyright (C) OpenMRS, LLC.
+ * All Rights Reserved.
+ */
+public interface ComplexViewHelper {
+	
+	public String getView(String view);
```

Comment:
This method should be JavaDoc'd.

Reply:
Let's provision for an `Obs` argument:
```java
public String getView(Obs obs, String view);
```
This will help the 2.x implementation support fetching the supported views for that obs, using for example: https://github.com/openmrs/openmrs-core/blob/7da5be1bc34fc4928779f303cd48d42b8a3cae0a/api/src/main/java/org/openmrs/api/ObsService.java#L417-L428

Reply:
Added javadoc, updated interface, updated calls to interfaces and formatting.